src/components/movieCard/MovieCard.jsx

```diff
@@ -0,0 +1,38 @@
+import "./MovieCard.css";
+import { useState } from "react";
+
+export function MovieCard({ title, description, posterURL, rating }) {
+  const [like, setlike] = useState("unlike");
+  function handleLike() {
+    let parent = document.getElementById("heartBtn");
+    let child = parent.querySelector("#heartIcn");
+
+    if (like === "like") {
+      setlike("unlike");
+      child.className = "fa-regular fa-heart";
+    } else {
+      setlike("like");
+      child.className = "fa-solid fa-heart";
+    }
+  }
+
+  return (
+    <div className="MovieCardContainer">
+      <p>{title}</p>
+      <p>{description}</p>
+      <p>{rating}</p>
+      <button
+        id="heartBtn"
```

Comment:
This id is not unique as the `MovieCard` component is rendered for each movie. Thus the logic above where you select by id will always return the first like button and thus the like functionality won't work as expected.